app/javascript/components/pages/JobsAuthenticate.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import SharedLayout from 'components/layout/SharedLayout';
+import Logo from 'components/icons/Logo';
+import Button from 'components/Button';
+
+import 'stylesheets/page';
+import 'stylesheets/jobs_authenticate';
+
+const JobsAuthenticate = () => {
```

Comment:
This is the new page! It's pretty straightforward. The plan is to redirect to this page when an unauthenticated person tries to access the job board. If they submit a valid password, they will be redirected to the job board.